tests/model_formsets_regress/tests.py

```diff
@@ -492,14 +492,14 @@ def test_custom_delete(self):
         data.update(self.delete_all_ids)
         formset = self.DeleteFormset(data, queryset=User.objects.all())
 
-        # verify two were deleted
+        # verify three were deleted
         self.assertTrue(formset.is_valid())
         self.assertEqual(len(formset.save()), 0)
-        self.assertEqual(len(User.objects.all()), 2)
+        self.assertEqual(len(User.objects.all()), 1)
 
-        # verify no "odd" PKs left
-        odd_ids = [user.pk for user in User.objects.all() if user.pk % 2]
-        self.assertEqual(len(odd_ids), 0)
+        # verify no odd serial values left
+        odd_serials = [user.serial for user in User.objects.all() if user.serial % 2]
+        self.assertEqual(len(odd_serials), 0)
```

Comment:
We could use `Mod()`, e.g.
```python
self.assertIs(
    User.objects.annotate(odd_serial=Mod('serial', 2)).filter(odd_serial=1).exists(),
    False,
)
```

Reply:
If you want. I think the current version is less complicated to mentally parse.